src/managed/CommonManaged.props

```diff
@@ -5,7 +5,24 @@
 
   <PropertyGroup>
```

Comment:
For my information. I assume this props file is essentially becoming our prop file for any SDK based projects.